swagger.yaml

```diff
@@ -87,8 +87,9 @@ definitions:
             $ref: '#/definitions/SubtopicsLink'
           content:
             $ref: '#/definitions/ContentLink'
+  # !!! Eleanor - should i now remove 'TopicResponse' ?
```

Comment:
You might need to ask Eleanor about this one 😄

Reply:
This will get caught and dealt with in future tickets.